docs/start/install.mdx

```diff
@@ -127,6 +128,52 @@ Tier 3 operating systems:
 - Linux i686 (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/) packaging specification) for Python 3.10 or later
 - Windows 32 bit for Python 3.10 or later
 
+
+## "No Module 'qiskit'" error with Jupyter Notebook
```

Comment:
could you add these 2 install faqs under a heading called "Troubleshooting" or something similar?

Reply:
Done!